os/fs/driver/mtd/smart.c

```diff
@@ -5581,6 +5598,15 @@ static int smart_journal_scan(FAR struct smart_struct_s *dev, bool print_dump)
 	} else {
 		dev->journal_seq = 0;
 	}
+	
+	for (int i = 0; i < dev->njournaleraseblocks; i++) {
+		/* Definately, below block should be erased */
+		fvdbg("current map : %d numberof journal : %d\n", i, dev->block_map[i]);
```

Comment:
How about printing the `block_map[i]` with unsigned format?
And I think adding space between `number` and `of` is better to read.

Reply:
Could you please change the line to: -
fvdbg("Current _Journal block_ : %d, number of _non-empty_ journals : %d\n", i, dev->block_map[i]);